add previously conflicting net/if.h include

Previously netlink/route/link.h included
linux/if.h, conflicting with net/if.h.
Starting with libnl 3.2.26 this include was removed.
So we have to include net/if.h now.

See: https://github.com/thom311/libnl/pull/73
or 50a76998ac36ace3716d3c979b352fac73cfc80a in libnl
--- a/python-ethtool/ethtool.c
+++ b/python-ethtool/ethtool.c
@@ -27,6 +27,7 @@
 #include <sys/types.h>
 #include <ifaddrs.h>
 #include <netlink/route/addr.h>
+#include <net/if.h>
 
 #include "etherinfo_struct.h"
 #include "etherinfo_obj.h"
